init.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
```

Comment:
This file should be removed, this is just for installing sphinx in the directory and can be deleted after that.

Reply:
Done.